packages/backend-for-frontend/src/utilities/eserviceDocumentUtils.ts

```diff
@@ -130,12 +130,12 @@ export const verifyAndCreateImportedDoc = async (
   // eslint-disable-next-line max-params
 ): Promise<void> => {
   const entry = entriesMap.get(doc.path);
-
-  const mimeType = mime.getType(doc.path) || "application/octet-stream";
-  if (entry === undefined) {
+  if (!entry) {
```

Comment:
I moved this check closer to the get, like we do in other services.

It's outside the scope of this PR but it's a very minor change.